### ISSUE TITLE
error when using thresreduce

### ISSUE THREAD

**Issue by ghoshaw** [KevinMusgrave/pytorch-metric-learning#465]
Hi, I use the loss as in readme:
loss_func = losses.TripletMarginLoss(distance = CosineSimilarity(), 
				     reducer = ThresholdReducer(high=0.3), 
			 	     embedding_regularizer = LpRegularizer())

but the error  is :
threshold_condition = low_condition & high_condition
TypeError: unsupported operand type(s) for &: 'bool' and 'Tensor' 
Do I need to change "high=torch.tensor(0.3) ? "

**Comment by KevinMusgrave** [comment]
What version of pytorch and pytorch-metric-learning are you using?

```python
import torch
import pytorch_metric_learning

print(torch.__version__)
print(pytorch_metric_learning.__version__)
```

**Comment by ghoshaw** [comment]
>>> print(torch.__version__)
1.7.1
>>> print(pytorch_metric_learning.__version__)
1.3.0

**Comment by KevinMusgrave** [comment]
This happens for pytorch versions < 1.10, so the quickest fix would be to upgrade to pytorch 1.10 or higher, if that's possible for you.

**Comment by ghoshaw** [comment]
ok , thanks

**Comment by KevinMusgrave** [comment]
Might as well fix this to work with older versions, so I'm re-opening until it's fixed.

**Comment by KevinMusgrave** [comment]
Fixed in v1.3.1
